Plugin/Block/Product/Image.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Fisheye Media Ltd. All rights reserved.
+ * See LICENCE.txt for licence details.
+ */
+namespace Fisheye\Lazyload\Plugin\Block\Product;
+
+
+class Image
```

Comment:
- [ ] Can we keep the class name / namespace as before please bar change from `Preference` to `Plugin`

i.e. `Fisheye\Lazyload\Plugin\OverrideProductImageTemplates` - it's best for plugins / preferences to explain their purpose rather than mirror the path of the class they are overriding